components/broadcast_talent_table/commons/broadcast_talent_table.lua

```diff
@@ -140,6 +140,10 @@ function BroadcastTalentTable:_fetchTournaments()
 		end
 	end
 
+	if args.isAchievementsTable then
+		conditions:add{ConditionNode(ColumnName('extradata_status'), Comparator.eq, '')}
```

Comment:
option1:
- add extradata_finished —> 'true'|'false' storage in sc2 infobox league
- purge all tournaments on sc2
- adjust consumers regarding using that instead of status field (shouldn't be many)
- remove the finished storage in status
- purge all tournaments on sc2 again
- only merge this pr after that is all done

option2:
- check for cancelled/postponed here

else you remove most of the caster achievements on sc2

example that i retrieved from a tier 1 event:
```json
{
  "result": [
    {
      "extradata": {
        "manualinput": "false",
        "status": "finished",
        "game": "lotv",
        "publishertier": "true",
        "liquipediatier": "1"
      },
      "wiki": "starcraft2"
    },
    .....
  ]
}
```

Reply:
## storage
- https://liquipedia.net/starcraft2/Module:HiddenDataBox/Custom
- https://liquipedia.net/starcraft2/Module:Infobox/League/Custom
- https://liquipedia.net/starcraft2/Module:Infobox/League/Hidden


## consumers
- https://liquipedia.net/starcraft2/Module:MainPageSeasonEvents
- https://liquipedia.net/starcraft2/Module:EPTPortalTournamentsList

Reply:
> Should we filter instead on not equals cancelled and not equals postponed
IMO specifically listing the ones to be excluded is preferrable